mod-audit-server/src/main/java/org/folio/dao/marc/impl/MarcAuditDaoImpl.java

```diff
@@ -72,6 +77,15 @@ public Future<List<MarcAuditEntity>> get(UUID entityId, SourceRecordType recordT
       .map(this::mapRowToAuditEntityList);
   }
 
+  @Override
+  public Future<Integer> count(UUID entityId, SourceRecordType recordType, String tenantId) {
+    LOGGER.debug("count:: Retrieving total count by tenantId: '{}', entityId: '{}', recordType '{}'", tenantId, entityId, recordType);
```

Comment:
Please double-check, as I remember the LOG strings already contain tenantId. For example (log4j2.properties):
appender.console.layout.pattern = %d{HH:mm:ss} [$${folio:requestid:-}] [$${folio:tenantid:-}]

Reply:
![image](https://github.com/user-attachments/assets/746b16c9-0425-4605-b4ec-af0cc247c77d)

Reply:
![image](https://github.com/user-attachments/assets/33be6cd8-bf12-4a0e-a27a-eb6622d98fe2)